client/boinc_cmd.cpp

```diff
@@ -188,6 +196,167 @@ void show_alerts(RPC_CLIENT &rpc) {
     }
 }
 
+////////////// --get_task_summary start ////////////////
+
+typedef vector<const char*> STR_LIST;
+
+// given: a list of lines, each consisting of n columns
+// for each column: find the longest entry.
+// Then display the lines so the columns line up.
+//
+void show_str_lists(vector<STR_LIST> lines, int ncols) {
+    vector<int> lengths;
+    int i;
+    for (i=0; i<ncols; i++) {
+        size_t max = 0;
+        for (STR_LIST& s: lines) {
+            max = std::max(max, strlen(s[i]));
+        }
+        lengths.push_back(max);
+    }
+    for (STR_LIST &line : lines) {
```

Comment:
since there is no plans to change the value of `len` I suggest to make it `const`